content/en/blog/2024/ambient-reaches-beta/index.md

```diff
@@ -0,0 +1,116 @@
+added, or the large consumption of resources by sidecars, and the inconvenience of the requirement to restart application pods
+with every new proxy version.
+
+As a community, we listened and designed ambient mode to tackle these problems. This new feature from Istio was named ambient
+mode as it was designed to be transparent to your application ensuring no additional configuration was required to adopt it
+and required no restarting of applications by users. You can refer to the [introducing ambient mesh blog](/blog/2022/introducing-ambient-mesh/)
+for more information on why we started ambient mode in Istio.
+
+## How does ambient mode make adoption easier?
+
+Istio’s ambient mode introduces lightweight, shared node proxies and optional Layer 7 (L7) proxies and removes the need for
+traditional sidecar proxies from the data plane. The core innovation behind ambient mode is that it slices the L4 and L7
+processing into two distinct layers. This layered approach allows you to adopt Istio incrementally, enabling a smooth
+transition from no mesh, to a secure overlay (L4), to optional full L7 processing — on a per-namespace basis, as needed across
+workloads.
+
+Ambient mode works without any modification required from your existing Kubernetes deployments. Users can label a namespace to
+add all of its workloads to Istio’s ambient mode, or opt-out of certain deployments as needed. By utilizing ambient mode, users
+bypass some of the previously restrictive elements of the sidecar model and instead can now expect server send-first protocols
+to work, see most of the reserved ports are now removed, and the ability for containers to bypass the sidecar – either
+maliciously or not – is greatly reduced.
+
+As part of the 1.22 release of Istio, the announcement of ambient mode in Beta also includes how this latest release brings
+both Layer 4 and Layer 7 mesh features to production readiness without sidecars.
+
+The lightweight shared L4 node proxy is called the zero-trust tunnel, or ztunnel. Ztunnel drastically reduces the overhead of
+running a mesh by removing the need to potentially over provision memory and CPU within a cluster to handle expected loads. In
+some use cases, the savings can exceed 90% or more, while still providing zero-trust security using mutual TLS with
+cryptographic identity, simple L4 authorization policies, and telemetry.
+
+The L7 proxies are called waypoints. Waypoints process L7 functions such as traffic routing, rich authorization policy
+enforcement, and enterprise-grade resilience. Waypoints run outside of your application deployments and can scale independently
+based on your needs, which could be for the entire namespace or for multiple services within the namespace. Compared with
+sidecars, you don’t need 1 waypoint proxy per application pod, and you can scale your waypoint effectively based on its scope,
```

Comment:
```suggestion
sidecars, you don’t need one waypoint per application pod, and you can scale your waypoint effectively based on its scope,
```

Reply:
... scale your waypoints based on the amount of traffic they handle.